### ISSUE TITLE
Sleeving Bug - Forced into sleeved character after observing with a new character during communicator calls

### ISSUE THREAD

**Issue by Xindii** [VOREStation/VOREStation#2052]
#### Brief description of the issue
My character got killed, so I respawned and observed as a new character, using a different slot. I fired up the ghost communicator to talk to someone, and got pulled out of it when the medical staff sleeved my old character, even after respawning to the menu and using a new slot.

#### What you expected to happen
I expected my old character's sleeve attempts to be fruitless, and my ability to use the ghost communicator as a new character.

#### What actually happened
I got pulled mid-comm call into the previous character. I left the round via teleporter, and respawned once again to try and continue the ghost-communicator conversation.
Ended up with me getting this error message when attempting to use the communicator as a ghost after respawning again. "Your identity is already present in the game world.  Please load in a different character first."

#### Steps to reproduce
- Get implanted.
- Die.
- Hit 'respawn'.
- Choose a new character slot.
- Smash that observe button.
- Fire up that communicator.
- Make pointless small talk to your comm-link target, and gripe about trivial things.
- Get medical to sleeve your previous character's body.
- Smash that dislike button when you are forced without an option, out of the communicator call and into your previous character.
- Leave the round.
- Use the call communicator verb.
- Seethe as you receive this error, which doesn't seem to disappear or resolve itself for ages. "Your identity is already present in the game world.  Please load in a different character first."

#### Code Revision
f4006f65d047af977cd5ba5e015caf706af8cb14 
 
#### Anything else you may wish to add:
Don't forget to smash those like buttons.

**Comment by kcin2001** [comment]
Use quit the round if you don't want to be resleeved

**Comment by Xindii** [comment]
I think you're misunderstanding the point. If I respawn and load up an entirely new slot, it should not put me back into the previous character.

**Comment by kcin2001** [comment]
Respawn doesnt do anything other than remove that ghost, use quit the round to fix your issue of being sleeved since it seems you didnt want to play that char anymore

**Comment by Natje0** [comment]
Yeah, I think it's pretty shitty to want the sleeving attempts to not work. Leave the round properly, and the issue won't exist in the first place.
Die and don't want to be resleeved? Use the Quit Round verb. Don't use round-about methods to try to make resleeving fail.
Pressing Respawn doesn't open your job slot either.

TL;DR Quit the Round verb solves literally any respawn-related problems, as it deletes your mind and body records, stops your Backup notif, opens your job slot, and removes you from the manifest.

**Comment by Xindii** [comment]
I was just reporting what I **thought** to be a bug. No need to swear or imply I want the whole sleeve-attempt thing to be changed.
